### PR TITLE
scuttle-shell

### PR DESCRIPTION
**Pull request by cryptix**
uses scuttle-shell@0.1 which is based on sbot@12

@mixmix this was based on your old branch. I'm not sure about two of the deps in the pkg.json I dragged back in (ssb-horcrux and style-resolve).

### CODE REVIEW

File: background-process.js [hunk 1]
```diff
@@ -1,34 +1,28 @@
...
-  .use(require('ssb-friends'))
-  .use(require('ssb-meme'))
-  .use(require('ssb-private'))
-  .use(require('ssb-query'))
-  .use(require('ssb-search'))
-  .use(require('ssb-unread'))
-  .use(require('ssb-ws'))
-  // .use(require('ssb-mutual')) // this is has recursion problems atm
+// Check if scuttle-shell is already running
+// TODO - make this check for scuttle-shell specifically (and not just an sbot)
```
Comment (cryptix) [inline background-process.js]: @mixmix I'm also not sure about this comment.

How would the shell be different from another sbot with the same plugins?

**Comment by christianbundy** [comment]
Is this branch still the best way to try scuttle-shell?

**Comment by mixmix** [comment]
yes but i've not tried it recently.

**Comment by cryptix** [comment]
I’m using it all the time and rebase it when I see something new intersting. ssb-about@2 should already work for instance.

**Comment by cryptix** [comment]
superseeded by #295

**Pull request closed without merging**